MUXSDKStats/MUXSDKStats/MUXSDKStats.m

```diff
@@ -260,5 +291,11 @@ + (void) orientationChangeForPlayer:(nonnull NSString *) name  withOrientation:(
     [player dispatchOrientationChange:orientation];
 }
 
++ (void)dispatchError:(nonnull NSString *)code withMessage:(nonnull NSString *)message forPlayer:(nonnull NSString *)name {
+    MUXSDKPlayerBinding *player = [_viewControllers valueForKey:name];
+    if (!player) return;
+    [player forceDispatchError:code withMessage:message];
```

Comment:
Oh I see the `forceDispatchError: withMessage:` method in `MUXSDKPlayerBinding` is supposed to always dispatch the error. I think having `force` in the name is fine, though I think it's equally clear to have the method be called:
`dispatchError:(nonnull NSString *)code withMessage:(nonnull NSString *)message`.

Reply:
good point!